.github/ISSUE_TEMPLATE/design-system-04-registration.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Component registration
```

Comment:
Maybe "New component registration" ? I had to scroll through to figure out exactly what this is.

Reply:
Great suggestion - thanks! I will add this change in next rev